gpx_csv_converter/__init__.py

```diff
@@ -59,6 +59,12 @@ def convert(self, gpx_string: str, output_file_name: str) -> None:
         for potential_field in potential_fields_hr:
             if len(mydoc.getElementsByTagName(potential_field)) > 0:
                 heart_rate_field = potential_field
+                
+        potential_fields_pwr = ["pwr:PowerInWatts"]
+        power_field = potential_fields_pwr[0]  # default
+        for potential_field_1 in potential_fields_pwr:
```

Comment:
probably could've used `potential_field` here, but IDK if reusing variables in this fashion is allowed in Python and if it would work